### ISSUE TITLE
Feature: Allow to disable compression

### ISSUE THREAD

**Issue by lucsoft** [cthackers/adm-zip#370]

**Comment by 5saviahv** [comment]
One method would be overwrite method directly:
``` js
const Zip = require("adm-zip");

// create file
const zip1 = new Zip({ noSort: true });
for (const file of ["a.txt", "c.txt", "b.txt", "a.txt"]) {
    zip1.addFile(file, file + " data");
}

// overwrite method: 0 = is STORED / 8 = is DEFLATE
zip1.getEntries().forEach((e) => {
    // ignore directory entries
    if (!e.isDirectory) {
        e.header.method = 0;
    }
});

// get file content
const result = zip1.toBuffer();
```

**Comment by lucsoft** [comment]
yeah this is what i use but its a bit hacky and i don't know if you load your compression code unnecessary

**Comment by lucsoft** [comment]
@5saviahv can the typescript types be not up-to-date as adding options to the constructor is not correct according to the types
![image](https://user-images.githubusercontent.com/11093382/115141956-6c94b180-a03f-11eb-9f3c-d17baa64ea6d.png)

**Comment by 5saviahv** [comment]
From v0.5.4 there are second parameter `options`, it is object type. It allows change some behaviour. Plan is add this disabling option also there. First parameter may also be `options` if we create new file (otherwise it should be undefined).

**Comment by lucsoft** [comment]
im on v5.5 but i use typescript and i don't have any options

and the types (@types/adm-zip) are on version 0.4.34 (latest)

i know there is a options field but its only for javascript

**Comment by 5saviahv** [comment]
I admit I have only few experiences with typescript, but I think maybe add type definitions into package. 

Other option would be make commits into `@types` repo, but it is huge and for developing you need probably entire thing.

**Comment by lucsoft** [comment]
typescript its awesome i recommend it and if its delivery in this package or even you write in typescript its a big improvement

**Comment by 5saviahv** [comment]
Issues #70 and #187 seems to related with this

**Comment by cthackers** [comment]
STORE compression can be set on constructor to disable compression
